pkg/cloud/ratelimit_test.go

```diff
@@ -82,3 +82,33 @@ func TestMinimumRateLimiter(t *testing.T) {
 		t.Errorf("`called` = true, want false")
 	}
 }
+
+func TestTickerRateLimiter(t *testing.T) {
+	t.Parallel()
+
+	trl := NewTickerRateLimiter(100, time.Second)
+	start := time.Now()
+	for i := 0; i < 50; i++ {
+		err := trl.Accept(context.Background(), nil)
+		if err != nil {
+			t.Errorf("TickerRateLimiter.Accept = %v, want nil", err)
+		}
+	}
+	elapsed := time.Since(start)
```

Comment:
It would be good to avoid timing-based tests as much as possible. What is the timing skew margin that this is sensitive to? We need to make it as robust to skew as possible.

Reply:
It should take a minimum of 500ms, so I check for the range [500ms, 1s]. I couldn't break. I think that mocking time or timer would make it unnecessarily complex while we only want to verify that `NewTickerRateLimiter` correctly converts its arguments to ticker period.